addons/project/static/src/project_sharing/views/registry.js

```diff
@@ -0,0 +1,10 @@
+/** @odoo-module **/
+
+import ViewRegistry from 'web.view_registry';
+
+import FormView from './form/view';
+import ListView from './list/view';
+
+ViewRegistry
+    .add('form', FormView)
```

Comment:
Normally, the value in the key is erased if the key already exists. 🙂 
The main goal I need to change the form and list views is to remove the action menu items to not allow the portal user to archive the tasks and also other actions in this menu.
Now, since the project sharing has own bundle, I could include the change rather than extend the form/list view.
What do you think ?